### ISSUE TITLE
[Bug]: Interactions-Addon still uses deprecated Button props

### ISSUE THREAD

**Issue by kartonrad** [storybookjs/storybook#26595]
### Describe the bug

We came across this after our Update to Storybook 8. 

This is the warning that is thrown
```
Use of deprecated props in the button component detected, see the migration notes at https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-ui-and-props-for-button-and-iconbutton-components 
```

This is an example of a line in this repository that uses the deprecated prop:
https://github.com/storybookjs/storybook/blob/7b9e094bc0f6645c6e955d98036ad7aa9b3acf51/code/addons/interactions/src/components/Subnav.tsx#L140
(MY particular warning also came from the rewind button, but i advise you to CTRL+F "containsIcon" to find all instances)

This is your migration guide for storybook 8:
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-ui-and-props-for-button-and-iconbutton-components 
There are other button props listed, that may still be used in places

### To Reproduce

Open the interactions panel when tests are running. 

### System

```bash
$ npx storybook@latest info

Storybook Environment Info:
(node:5204) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

  System:
    OS: Windows 10 10.0.19045
    CPU: (24) x64 12th Gen Intel(R) Core(TM) i9-12950HX
  Binaries:
    Node: 21.1.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 4.0.1 - C:\Program Files\nodejs\yarn.CMD <----- active
    npm: 8.19.2 - ~\Documents\XXXXXXXXX\node_modules\.bin\npm.CMD
  Browsers:
    Edge: Chromium (122.0.2365.92)
  npmPackages:
    @storybook/addon-coverage: ^1.0.1 => 1.0.1
    @storybook/addon-essentials: ^8.0.2 => 8.0.2
    @storybook/addon-interactions: ^8.0.2 => 8.0.2
    @storybook/addon-links: ^8.0.2 => 8.0.2
    @storybook/addon-webpack5-compiler-babel: 3.0.3 => 3.0.3
    @storybook/blocks: ^8.0.2 => 8.0.2
    @storybook/react: ^8.0.2 => 8.0.2
    @storybook/react-webpack5: ^8.0.2 => 8.0.2
    @storybook/test: 8.0.2 => 8.0.2
    @storybook/test-runner: ^0.17.0 => 0.17.0
    eslint-plugin-storybook: ^0.8.0 => 0.8.0
    storybook: ^8.0.2 => 8.0.2
```

**Comment by vtsvetkov-splunk** [comment]
I got the same. I supposed it should be fixed there #25760 but only got fixed for icons, not for addon-interactions

**Comment by zz-james** [comment]
Haven't fully investigated but I am getting the warning
![image](https://github.com/storybookjs/storybook/assets/1120870/0239277a-000b-4240-a374-fd191cb7d78f)
